src/traccuracy/track_errors/_ctc.py

```diff
@@ -91,15 +90,19 @@ def get_edge_errors(matched_data: "Matched"):
     )
     gt_graph.set_edge_attribute(list(gt_graph.edges()), EdgeAttr.FALSE_NEG, False)
 
-    node_mapping_first = np.array([mp[0] for mp in node_mapping])
-    node_mapping_second = np.array([mp[1] for mp in node_mapping])
+    node_mapping_first = {
```

Comment:
nit: would love to rename this to 'gt_comp_mapping' and `comp_gt_mapping` respectively

Reply:
Very much agree, updated now.